model/task_lifecycle.go

```diff
@@ -1868,6 +1902,17 @@ func UpdateBuildAndVersionStatusForTask(ctx context.Context, t *task.Task) error
 		if err = checkUpdateBuildPRStatusPending(ctx, taskBuild); err != nil {
 			return errors.Wrapf(err, "updating build '%s' PR status", taskBuild.Id)
 		}
+		// only do it for version, patches need wait for child patches
```

Comment:
nit: "need to wait" but if you're keeping this comment may also be worth cleaning up ("add tracing" instead of "do it")